decisions/2021/01/14-frontend-initiatives.md

```diff
@@ -0,0 +1,137 @@
+
+      - Scheduler App - [@ismay](https://github.com/ismay)
+
+        Discussion: At the moment isn't ready to go in. Wahed has been the one working worked on it, whom is no longer part of dhis2. However as of now, most of the blockers are resolved. Ismay willl take over this work. In 30 days we arent 100 percent sure if it can be ready for review.
+        Austin will talk with Wahed and discuss about the app, this will be sort of handover. Also Hendrik suggested he would like to be included.
+
+      - Settings App - [@amcgee](https://github.com/amcgee)
+
+      - Maps App (blocked by Plugin builds, no feature toggling) - [@amcgee](https://github.com/amcgee) [@turban](https://github.com/turban)
+
+        Discussion: Austin and Bjorn gave progress update.
+
+      - Update deps in all apps - [@amcgee](https://github.com/amcgee)
+
+        Discussion: Austin gave progress update.
+
+    - Continuous Delivery & App Hub updates - [@Birkbjo](https://github.com/Birkbjo) [@amcgee](https://github.com/amcgee)
+
+      Discussion: Birk gave update on the topic. There is wip on the topic. 
+
+      Two things we want to look into is a) we want to be able to enable continuous delivery. b) We want to have the authentication with the backend in place.
+
+    - Extra topic 
+      Discussion: Hendrik and Austin will connect with Ulaf regarding the data quality app.
+
+* Updates from active working groups
+
+  - Cypress - [@HendrikThePendric](https://github.com/HendrikThePendric) [@varl](https://github.com/varl) (out Jan) [@Mohammer5](https://github.com/Mohammer5) (out Jan)
+
+    Discussion: Hendrik is about to release the network shim. New version of cypress-cli v5 is released. 
+
+  - PWA & Offline/Responsive Dashboard - (see above)
+    - PWA
+    
+        Discussion: Austin had an mvp on that that he tested with some people on their apps.
+
+    - Responsive Dashboard
+    
+        Discussion: Jan is making progress on the work. Milagros and Jeniffer are the main people working on it.
+        Jeniffer works on the responsivness of the app. This part is in PR. Overall is looking pretty good. 
+        Milagros, is work in progress. Couple of details is left.
+
+    - Offline design update from Joe
+    
+        Discussion: Joe has a design proposal for offline connectivity for mobile and desktop. The concept is there and it is basically ready to be implemented. Mobule version is 90 percent there.
+        There will be an upcoming meeting with Jan, Joe
+
+  - UI & Platform updates - [@ismay](https://github.com/ismay) [@HendrikThePendric](https://github.com/HendrikThePendric) [@cooper-joe](https://github.com/cooper-joe) [@amcgee](https://github.com/amcgee)
+
+    * ui cons 
+
+      Discussion: are released and people can use it. There is also a strategy for migration that Joe has worked on. There is a breaking change to ui v5, the change is that if you have been using the old icons you want to be careful because the old icons have been removed.
```

Comment:
The ui-icons migration strategy is available for dashboards, data visualizer and (soon) maps. Other apps do not have a migration strategy available yet.